src/components/atoms/OakFieldset/OakFieldset.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+
+import { colorStyle, ColorStyleProps } from "@/styles/utils/colorStyle";
+import { opacityStyle, OpacityStyleProps } from "@/styles/utils/opacityStyle";
+import {
+  marginStyle,
+  MarginStyleProps,
+  paddingStyle,
+  PaddingStyleProps,
+} from "@/styles/utils/spacingStyle";
+import {
+  typographyStyle,
+  TypographyStyleProps,
+} from "@/styles/utils/typographyStyle";
+import { borderStyle, BorderStyleProps } from "@/styles/utils/borderStyle";
+import { sizeStyle, SizeStyleProps } from "@/styles/utils/sizeStyle";
+
+export type OakFieldsetProps = ColorStyleProps &
+  SizeStyleProps &
+  OpacityStyleProps &
+  MarginStyleProps &
+  PaddingStyleProps &
+  BorderStyleProps &
+  TypographyStyleProps;
+
+/**
+ * OakFieldset renders a custom `fieldset` component, removes default styling of fieldset.
+  color, opacity, margin, padding, border and typography styles can be passed in also.
+ */
+export const OakFieldset = styled.fieldset<OakFieldsetProps>`
+  border: 0px;
+  margin: 0;
+  padding: 0;
+  ${colorStyle}
+  ${sizeStyle}
+  ${opacityStyle}
+  ${marginStyle}
+  ${paddingStyle}
+  ${borderStyle}
+  ${typographyStyle}
+`;
```

Comment:
I think this should have the same props as `OakBox` if you look at `OakForm` it imports the styles and props from `OakBox` so you don't have to compose all the styles yourself.

Something like this 👇🏻 
```suggestion
import styled from "styled-components";

import { OakBoxProps, oakBoxCss } from "@/components/atoms/OakBox";

export type OakFieldsetProps = OakBoxProps;
/**
 * OakFieldset renders a custom `fieldset` component, removes default styling of fieldset.
  color, opacity, margin, padding, border and typography styles can be passed in also.
 */
export const OakFieldset = styled.fieldset<OakFieldsetProps>`
  border: 0;
  margin: 0;
  padding: 0;
  ${oakBoxCss};
`;

```

Reply:
Not sure if you've watched The Bear (recommended if not)
<img src="https://media0.giphy.com/media/EANve5CkUkG4w089vw/giphy.gif"/>

Reply:
Not sure if you've played Overcooked https://www.tiktok.com/@mitchell_ferguson/video/7324905711157202218